### PR TITLE
feat(breadcrumbs): add prop to display trailing separator

### PR DESCRIPTION
**Pull request by shannonmoeller**
#### Description

Adds the option to show a trailing separator after the last breadcrumb.

#### Scope

Minor

### CODE REVIEW

File: src/breadcrumbs/__tests__/breadcrumbs.test.js [hunk 1]
```diff
@@ -38,6 +38,19 @@ describe('Breadcrumbs', () => {
     ).toMatchSnapshot();
   });
 
+  it('displays trailing separator', () => {
```
Comment (gergelyke) [inline src/breadcrumbs/__tests__/breadcrumbs.test.js:41]: can we add a visual regression test instead? something like this: https://github.com/uber/baseweb/blob/master/src/breadcrumbs/__tests__/breadcrumbs.scenario.js

**Comment by gergelyke** [comment]
thank you Shannon! one more thing: would you mind also adding it to the react-view config? https://github.com/uber/baseweb/blob/master/documentation-site/components/yard/config/breadcrumbs.ts

File: src/breadcrumbs/__tests__/breadcrumbs.test.js (hunk 1 above)
Comment (shannonmoeller) [reply src/breadcrumbs/__tests__/breadcrumbs.test.js:41]: You bet

**Comment by shannonmoeller** [comment]
On it.

**Pull request merged**